persistence-modules/core-java-persistence/src/main/java/com/baeldung/jdbcmetadata/MetadataExtractor.java

```diff
@@ -0,0 +1,118 @@
+package com.baeldung.jdbcmetadata;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class MetadataExtractor {
+	private final DatabaseMetaData databaseMetaData;
+
+	public MetadataExtractor(Connection connection) throws SQLException {
+		this.databaseMetaData = connection.getMetaData();
+        DatabaseMetaData databaseMetaData = connection.getMetaData();
+	}
+	
+	public void extractTableInfo() throws SQLException {
+        ResultSet resultSet = databaseMetaData.getTables(null, null, "CUST%", new String[]{"TABLE"});
+        while(resultSet.next())
+        {
```

Comment:
Opening braces should be at the end of a line of code, not at the beginning of a line by itself. Several of the while statements are done like this.